dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/sanitizer/UriParsingException.kt

```diff
@@ -0,0 +1,6 @@
+package com.datadog.gradle.plugin.internal.sanitizer
+
+import java.rmi.RemoteException
+
+internal class UriParsingException(message: String, throwable: Throwable) :
```

Comment:
I'm not sure a `RemoteException` is the appropriate parent class for this (especially coming from the `java.rmi` package : 
>  A `RemoteException` is the common superclass for a number of communication-related exceptions that may occur during the execution of a remote method call

If we do need a custom exception, I'd make it a child of either `java.net.MalformedURLException` or `java.lang.IllegalArgumentException`.

Reply:
Yes...as I explained to @0xnm this was a mistake, my intention was to use a `RuntimeException` is just the editor playing tricks on me.